dlib/dnn/loss.h

```diff
@@ -993,6 +993,36 @@ namespace dlib
         }
     }
 
+    std::ostream& operator<<(std::ostream& out, const std::vector<mmod_options::detector_window_details>& detector_windows)
```

Comment:
Global functions like this need to be declared inline if they are in header files, or linker errors will result if they are #included into more than one translation unit.

Reply:
Fixed.